src/AzureClient/Magic/AzureClientMagicBase.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Jupyter.Core;
+using Microsoft.Quantum.IQSharp.Common;
+using Microsoft.Quantum.IQSharp.Jupyter;
+
+namespace Microsoft.Quantum.IQSharp.AzureClient
+{
+    /// <summary>
+    ///     Base class used for Azure Client magic commands.
+    /// </summary>
+    public abstract class AzureClientMagicBase : AbstractMagic
+    {
+        /// <summary>
+        ///     The object used by this magic command to interact with Azure.
+        /// </summary>
+        public IAzureClient AzureClient { get; }
+
+        /// <summary>
+        ///     Constructs the Azure Client magic command with the specified keyword
+        ///     and documentation.
```

Comment:
I don't understand from this description what is the keyword here - can you please elaborate? 
same for documentation.

Reply:
Yes, the keyword is used to invoke the magic command (e.g. 'connect'), and the documentation describes how to use the magic command. Sure, I'll elaborate a bit here.

---
In reply to: [426993988](https://github.com/microsoft/iqsharp/pull/144#discussion_r426993988) [](ancestors = 426993988)